io_scene_xray/prefs/props.py

```diff
@@ -247,9 +249,10 @@ def _auto_path(prefs, self_name, path_suffix, checker):
             fs = rw.ltx.LtxParser()
             fs.from_file(prefs.fs_ltx_file)
         except:
+            traceback.print_exc()
```

Comment:
Would write logs like:
<img width="980" alt="image" src="https://github.com/PavelBlend/blender-xray/assets/5264687/51dd8a0e-717e-479d-a7ee-484f1cc70cd7">

Not sure if it's good though